doc/source/build_hints.rst

```diff
@@ -1803,11 +1803,86 @@ The following options are available to select a subset of drivers:
     Independently of options that control global behavior, drivers can be individually
     enabled or disabled with those options.
 
+    .. note::
+
+        <driver_name> above and below is *generally*, but not systematically the short driver name.
+
+        Some drivers may also be grouped together for build purposes.
+
+        - A number of "raw" raster drivers (ACE2, BT, BYN, CPG, CTable2, DIPEx, DOQ1,
+          DOQ2, EHDR, EIR, ENVI, FAST, GenBIN, GSC, GTX, MFF2, ISCE, KRO, MFF, LAN,
```

Comment:
I don't see GTX listed in https://gdal.org/drivers/raster/index.html, was it removed in master?

Reply:
no the driver is still there, but lacks a documentation page. #5820 created about that